ports/espressif/boards/mixgo_ce_udisk/mpconfigboard.mk

```diff
@@ -0,0 +1,24 @@
+USB_VID = 0x239A
+USB_PID = 0x80A8
```

Comment:
You'll need your own USB VID/PID combination. Espressif usually gives them out here: https://github.com/espressif/usb-pids